aii-core/src/main/perl/aii-web.cgi

```diff
@@ -9,11 +9,14 @@
 # is missing the node will boot from the hard disk
 my $boothd = "default";
 
-# Quattor XML profiles
+# Quattor profiles
 my $web_root = "/var/www/html";
```

Comment:
@samary ideally, the `package aii_shellfev2;` from `aii-shellfe` is a proper module, and we could make a fake application instance, which would then get automatic access to the configfiles and the default values.
but it would require a refactor of the `aii-shellfe`, and not everyone might agree with this idea.
so leave it for now i think. i won't object the current code :smile: